src/containers/Navigation/Navigation.vue

```diff
@@ -71,8 +71,8 @@ const navbarBlacklist = [
   'Fluency-ARF-ES',
   'Fluency-CALF',
   'Fluency-CALF-ES',
-  'Fluency-Alpaca',
-  'Fluency-Alpaca-ES',
+  'Roam-Alpaca',
+  'Roam-Alpaca-ES',
```

Comment:
Does it break anything if we capitalize these? If so, can we change the route names everywhere to capitalize "ROAM?"

```suggestion
  'ROAM-Alpaca',
  'ROAM-Alpaca-ES',
```